pom.xml

```diff
@@ -23,7 +23,7 @@
     <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
     
     <!-- Fuse version -->
-    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
```

Comment:
It could be done in a separate PR, but like the changes for the BPMN2 modeler these properties don't change a whole heck of a lot. If we document it in the JIRA is that enough? Seems like a lot of overkill to do a whole separate PR